### PR TITLE
Release candidate 2.0

### PR DESCRIPTION
**Pull request by Nestastnikos**
Changes:
- the number of scenarios per system call has been expanded
- the way how the test cases are implemented changed, now they are implemented as test class methods
- added test encapsulation
- added configurable logger
- added support for running tests on offstream branches
- changed running of tests from GUI to CLI
- configuration of testing execution is done with YAML config
- added support for multiple authorization servers (if available)
- removed support for concurrency tests
- dropped support for generating HTML reports (for now)
- split guest scripts and host scripts into separate folders

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,4 +37,3 @@ def run(self):
       zip_safe=True,
       include_package_data=True,
       cmdclass={'install': MyInstall})
-
```
Comment (Programator2) [inline setup.py:40]: Change version to 2.0.0-rc.1

File: setup.py [hunk 2]
```diff
@@ -25,15 +25,15 @@ def run(self):
     subprocess.call(['py', '-2', 'vboxapisetup.py', 'install'])
     os.chdir(old_path)
 
-setup(name='mte',
-      version='0.1.0',
-      description='Testing environment for Medusa security system',
-      url='https://github.com/Programator2/medusa-testing-environment',
-      author='Roderik Ploszek',
+setup(name='mits',
+      version='2.0.0',
```
Comment (Programator2) [inline setup.py]: So, this is not a release candidate, but a final version?

File: setup.py [hunk 3]
```diff
@@ -25,15 +25,15 @@ def run(self):
...
+      description='Testing integration system for Medusa security system',
+      url='https://github.com/medusa-team/medusa-testing-environment',
+      author='Team Medusa',
```
Comment (Programator2) [inline setup.py]: Use a person's name. Me or you, I'll let you decide. And leave/change email accordingly.

**Comment by Programator2** [comment]
Any update on my comments?

File: setup.py (hunk 3 above)
Comment (Nestastnikos) [reply setup.py]: I left there your name.

**Pull request merged**